ember-flight-icons/README.md

```diff
@@ -37,16 +37,16 @@ The `name` must be specified. A default icon size of 24 will be provided.
 
 ### Optional usage
 
-Icons are set to a `viewBox` square size of 24 by default.
```

Comment:
there's a reference to default of 24px up on line 32 ☝️  that we should update to 16

Reply:
Am I wrong, or do I remember yesterday we discussed this and we agreed to make the size mandatory (without a default) and not have a predefined size at code level, only in Figma?

Reply:
@didoo I remember discussing it as an option, but I didn't think we'd made a hard decision on it. So far it seems like we're trying to match the current developer experience, which seems to be to provide strong defaults with customizable options for flexibility when needed. I wonder if changing that approach would mean a change via RFC? What do you think?

Reply:
yesterday, i thought we agreed to change size to 16. should we consider the size being required in a separate issue? getting this in for the smoke testing folks are doing in consuming apps rn seems prudent?